cmd/promtool/unittest.go

```diff
@@ -91,7 +104,8 @@ func ruleUnitTest(filename string, queryOpts promqltest.LazyLoaderOpts, run *reg
 	}
 
 	evalInterval := time.Duration(unitTestInp.EvaluationInterval)
-
+	currentTime := time.Now().Format("2006-01-02T15:04:05")
+	ts.Settime(currentTime)
```

Comment:
I think this could be one line and wouldn't hurt readability, no need for a variable.